Src/WitsmlExplorer.Frontend/contexts/navigationStateReducer.tsx

```diff
@@ -90,6 +92,10 @@ const selectToggleTreeNode = (state: NavigationState, { payload }: ToggleTreeNod
   };
 };
 
+const collapseAllTreeNodes = (state: NavigationState): NavigationState => {
+  return { ...state, ...allDeselected, expandedTreeNodes: [], selectedServer: state.selectedServer, currentSelected: state.selectedServer };
```

Comment:
I suggest either changing the name so it indicates that all elements will be deselected, or including a boolean in the payload so the programmer can decide if they should be deselected or not when calling the dispatch.

Reply:
This has been removed because the `selectServer` method can achieve the same result.